axelor-account/src/main/resources/views/Move.xml

```diff
@@ -195,6 +195,8 @@
 		<view type="grid" name="move-line-grid" />
 		<view type="form" name="move-line-form" />
 		<view-param name="search-filters" value="move-line-filters"/>
+		<domain>self.move.accountingReport.id = :accountingReport_id</domain>
+		<context name="accountingReport_id" expr="eval: _accountingReportId"/>
```

Comment:
This will create issue when click from normal Move grid (from menu), as that time no 'accountingReportId' will be there.